test/integration/test-amp-analytics.js

```diff
@@ -190,6 +191,155 @@ describe.configure().skipIfPropertiesObfuscated().run('amp' +
+        // see vendors/googleanalytics.js "pageview" request for config
+        expect(queries).to.include({
+          _v: 'a1',
+          _r: '1',
+          v: '1',
+          cid: '1427830804.1524174812',
+          dr: '',
+          ds: 'AMP',
+          dt: 'AMP TEST',
+          tid: 'UA-67833617-1',
+          t: 'pageview',
+        });
+        const isNumber = /^\d+$/;
+        const isRandomNumber = /^0\.\d+$/;
+        expect(queries['dl']).to.contain('/amp4test/compose-doc?'); // ${documentLocation}
+        expect(queries['_s']).to.match(isNumber); // ${requestCount}
+        expect(queries['_utmht']).to.match(isNumber); // ${timestamp}
+        expect(queries['sr']).to.match(/^\d+x\d+$/); // ${screenWidth}x${screenHeight}
+        expect(queries['sd']).to.match(isNumber); // ${screenColorDepth}
+        expect(queries['ul']).to.be.ok; // ${browserLanguage}
+        expect(queries['de']).to.be.ok; // ${documentCharset}
+        expect(queries['jid']).to.match(isRandomNumber); // ${random}
+        expect(queries['a']).to.match(isNumber); // ${pageViewId}
+        expect(queries['z']).to.match(isRandomNumber); // ${random}
+      });
+    });
+  });
+
+  describes.integration('amp-analytics type=googleanalytics new user', {
+    body: `
+      <script>
+        // expires existing _ga cookie if any
+        document.cookie='_ga=;expires=' + new Date(0).toUTCString();
+      </script>
+      <amp-analytics type="googleanalytics">
+        <script type="application/json">
+        {
+          "vars": {
+            "account": "UA-67833617-1"
+          },
+          "requests": {
+            "host": "${RequestBank.getUrl(1)}"
+          },
+          "triggers": {
+            "pageview": {
+              "on": "visible",
+              "request": "pageview"
+            },
+            "performanceTiming": {
+              "enabled": false
+            },
+            "adwordsTiming": {
+              "enabled": false
+            }
+          }
+        }
+        </script>
+      </amp-analytics>
+      <amp-analytics type="googleanalytics">
+        <script type="application/json">
+        {
+          "vars": {
+            "account": "UA-67833617-1"
+          },
+          "requests": {
+            "host": "${RequestBank.getUrl(2)}"
+          },
+          "triggers": {
+            "pageview": {
+              "on": "visible",
+              "request": "pageview"
+            },
+            "performanceTiming": {
+              "enabled": false
+            },
+            "adwordsTiming": {
+              "enabled": false
+            }
+          }
+        }
+        </script>
+      </amp-analytics>
+      `,
+    extensions: ['amp-analytics'],
+  }, () => {
+    afterEach(() => {
+      // clean up written _ga cookie
+      document.cookie = '_ga=;expires=' + new Date(0).toUTCString();
+    });
+
+    it('should assign new cid', () => {
```

Comment:
Do we have test to get stored cid? Or is that not possible to do.

Reply:
the test above is parsing CID from cookie

Reply:
Missed that. Thanks.